tests/scripts/utils.sh

```diff
@@ -1,7 +1,20 @@
+JUNIT_SETUP_TEMPLATE=$(cat << EOF
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+    <testsuite name="SUITE_ID">
+        <testcase name="setup" classname="SUITE_ID.setup">
+            <failure message="OLS failed to start up for SUITE_ID"/>
+        </testcase>
+    </testsuite>
+</testsuites>
+EOF
+)
+
+
 # Wait for the ols api server to respond 200 on the readiness endpoint
 # $1 - url of the ols server to poll
 function wait_for_ols() {
-  for i in {1..30}; do
+  for i in {1..60}; do
```

Comment:
i'm also raising this so that we can be beyond all doubt that it's just a timeout problem.